DESCRIPTION

```diff
@@ -71,7 +71,7 @@ Suggests:
     cplm,
     dplyr,
     DRR,
-    effectsize (>= 0.4.0),
+    effectsize (>= 0.4.0.001),
```

Comment:
I would rather leave this as is (i.e. >= 0.4.0), and then check the package version in the test file (i.e. run tests conditionally if package version of effectsize is > 0.4.0), else there might be troubles when updating / loading the package, using _easystats_ etc.